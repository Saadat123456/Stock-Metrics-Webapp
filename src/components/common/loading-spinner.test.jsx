import { render } from '@testing-library/react';
import LoadingSpinner from './loading-spinner';

test('Renders Correctly', () => {
  const { container } = render(
    <LoadingSpinner />,
  );
  expect(container.firstChild).toMatchSnapshot();
});
