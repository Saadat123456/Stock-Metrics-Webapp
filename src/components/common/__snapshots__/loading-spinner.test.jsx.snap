// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<div
  class="lds-ripple"
  data-testid="loading"
>
  <div />
  <div />
</div>
`;
